docs/genai/howto/install.md

```diff
@@ -23,61 +29,149 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
 
 Ensure that the `CUDA_PATH` environment variable is set to the location of your CUDA installation.
 
+
 ## Python packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these sets of packages (CPU, DirectML, CUDA) should be installed in your environment.
 
 ### CPU
 
+#### x64
+
 ```bash
-pip install numpy
+# Install ONNX Runtime nightly. Update this to the released version when it is available.
+pip install ort-nightly --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
+
+pip install onnxruntime-genai --pre
+```
+
+#### Arm64
```

Comment:
Break down into windows and linux. For arm64 on linux, we still depend on ort-nightly

Reply:
Reverted the Python install updates for now. Will refer to this when we update